mm/tlsf/0006-tlfs-fix-link-failed.patch

```diff
@@ -0,0 +1,31 @@
+From 24a394b8cea3198647b45feae8cf6b875b5bab8f Mon Sep 17 00:00:00 2001
+From: Jiuzhu Dong <dongjiuzhu1@xiaomi.com>
+Date: Fri, 19 Aug 2022 23:03:42 +0800
+Subject: [PATCH 6/8] tlfs: fix link failed
+
+VELAPLATFO-3223
+
+nuttx/mm/tlsf/tlsf/tlsf.c:1023: undefined reference to `merr'
+/usr/bin/ld: /home/djz/workspace/test/sim/nuttx/mm/tlsf/tlsf/tlsf.c:1032: undefined reference to `merr'
+
+Change-Id: I7c59c623ef47a4479adfff5e37c2825ab9d3db78
+Signed-off-by: Jiuzhu Dong <dongjiuzhu1@xiaomi.com>
+---
+ tlsf.c | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/tlsf.c tlsf/tlsf/tlsf.c
+index 536bdff..75c7129 100644
+--- a/tlsf.c
++++ tlsf/tlsf/tlsf.c
+@@ -4,6 +4,7 @@
+ #include <stdio.h>
+ #include <stdlib.h>
+ #include <string.h>
++#include <debug.h>

```

Comment:
let's map tlsf_printf to NULL and remove mm/tlsf/0006-tlfs-fix-link-failed.patch